Source/utils/ini.cpp

```diff
@@ -74,6 +80,22 @@ Ini::Values::Values(const Value &data)
 {
 }
 
+std::vector<std::string> Ini::getKeys(std::string_view section) const
+{
+	const auto sectionIt = data_.sections.find(section);
+	if (sectionIt == data_.sections.end()) return {};
+
+	std::vector<std::pair<std::string, ValuesData>> entries;
+	entries.assign(sectionIt->second.entries.begin(), sectionIt->second.entries.end());

```

Comment:
This can be done as a one-liner:

```c++
std::vector<std::pair<std::string, ValuesData>> entries(sectionIt->second.entries);
```

If the above doesn't work:

```c++
std::vector<std::pair<std::string, ValuesData>> entries(sectionIt->second.entries.begin(), sectionIt->second.entries.end());
```